sdks/java/extensions/protobuf/build.gradle

```diff
@@ -18,8 +18,13 @@
 
 plugins { id 'org.apache.beam.module' }
 applyJavaNature(
-  enableChecker: false,
   ignoreRawtypeErrors: true,
+  generatedClassPatterns: [
+    '^org\\.apache\\.beam\\.sdk\\.extensions\\.protobuf\\.Proto2CoderTestMessages',
+    '^org\\.apache\\.beam\\.sdk\\.extensions\\.protobuf\\.Proto2SchemaMessages',
+    '^org\\.apache\\.beam\\.sdk\\.extensions\\.protobuf\\.Proto3SchemaMessages',
+    '^org\\.apache\\.beam\\.sdk\\.extensions\\.protobuf\\.Proto3SchemaOptions',
```

Comment:
Is it possible to use [slashy strings](https://groovy-lang.org/syntax.html#_slashy_string) in our gradle files? That'd clean these arrays up a bit.

Reply:
Didn't know about slashy strings. Nice. Done.

Reply:
Nice! I was unaware of them before, but did a check to see if there was some kind of regex literal we could use. I guess `~"pattern"` is an option as well, but produces `Pattern` instances.